docs/protocol/network/data-structures/transaction.md

```diff
@@ -0,0 +1,136 @@
+# Transaction
+
+In the Witnet network protocol, a `transaction` is formatted as follows:
+
+| Field     |    Type    | Description                                                               |
+| --------- | :--------: | ------------------------------------------------------------------------- |
+| `version` |   `u32`    | The transaction data format version number                                |
+| `inputs`  | `[input]`  | A list of 1 or more transaction inputs                                    |
+| `outputs` | `[output]` | A list of transaction outputs                                             |
+| `claims`  | `[claim]`  | A list of 1 or more claims (i.e. spending conditions), one for each input |
+
+The inputs and claims lists cannot be empty as trasactions require at least to consume an unspent output.
+
+Signature structures are defined in the [Signature] section.
+
+## Inputs (aka outpoints)
+
+The transaction inputs are also known as outpoints because they refer to an existing unspent transaction output.
```

Comment:
Transaction inputs don't necessarily point to unspent transaction outputs. That's only truth for inputs in transactions that are not yet anchored to the chain and that don't point to outputs in other transactions that are equally not yet anchored to the chain.

This paragraph could read:
```suggestion
Transaction inputs are references to outputs from past transactions. They "pull" all the value from those outputs and make it available for being spent by the outputs in the same transaction they are in. This data structure—which pairs a transaction's identifier with the index of one of its outputs—is also known as `outpoint` as it unambiguously points to a unique output from a specific transaction.
```